sspl_test/plans/self_secondary.pln

```diff
@@ -1,3 +1,3 @@
 alerts.self_hw.test_self_hw_real_stor_enclosure_conn
 alerts.node.test_node_fan_actuator
-alerts.self_hw.test_self_hw_node_sel_event
\ No newline at end of file
+alerts.self_hw.test_self_hw_node_sel_event
```

Comment:
no change here, file should not reflect in PR

Reply:
removed.